docs/dev/topology-over-raft.md

```diff
@@ -446,7 +446,6 @@ CREATE TABLE system.topology (
     new_cdc_generation_data_uuid timeuuid static,
     new_keyspace_rf_change_ks_name text static,
     new_keyspace_rf_change_rf_per_dc frozen<map<text, text>> static,
-    // TODO: this schema is heavily outdated, should I update it with the remaining columns?
```

Comment:
seems unrelated to current patch

Reply:
Yes, and as with https://github.com/scylladb/scylladb/pull/16723#discussion_r1591101076, what to do about it?